mono/metadata/decimal-ms.c

```diff
@@ -1119,14 +1119,14 @@ DecAddSub(MonoDecimal *left, MonoDecimal *right, MonoDecimal *result, int8_t sig
 }
 
 // Decimal addition
-static MonoDecimalStatus
+static MonoDecimalStatus G_GNUC_UNUSED
```

Comment:
I guess at that early point in the fixing I wasn't hard-boiled enough yet to believe that those function  are still there just by accident.